### ISSUE TITLE
ATTENTION!!

### ISSUE THREAD

**Issue by LankyBox01** [PigeonVideo/PigeonVideo.github.io#8]
I got banned on glitch support for a few weeks, so we will talk here.
Post this to the group DM

**Comment by Readtt** [comment]
f in the chat

**Comment by LankyBox01** [comment]
Lol

**Comment by LankyBox01** [comment]
Let's use socket.io

**Comment by Readtt** [comment]
Yes, Im a bit of a trash pro at socket.io

**Comment by LankyBox01** [comment]
https://pigeon-video-chat.glitch.me/

**Comment by LankyBox01** [comment]
Please tell evertyone in the group DM to enter

**Comment by Readtt** [comment]
I think I can make it look a bit better ngl

**Comment by LankyBox01** [comment]
I didn't make it

**Comment by Readtt** [comment]
oh, ok

**Comment by LankyBox01** [comment]
Hi

**Comment by LankyBox01** [comment]
Hello?

**Comment by LankyBox01** [comment]
Were you H?

**Comment by LankyBox01** [comment]
If you were, i was Hack-r, i just pretended to be a hacker.
I opened 2 windows and prentended that i made a bot, when i was just typing nonsense.

**Comment by LankyBox01** [comment]
No actual hacking involved.

**Comment by greasezone** [comment]
Hello!

**Comment by greasezone** [comment]
I am at the right place?

**Comment by LankyBox01** [comment]
Yup

**Comment by greasezone** [comment]
Want to chat on glitch?

**Comment by LankyBox01** [comment]
I can't
I got banned

**Comment by LankyBox01** [comment]
:(

**Comment by greasezone** [comment]
Sorry! What happened? Are you banned forever?

**Comment by aboutDavid** [comment]
Until next week

**Comment by LankyBox01** [comment]
No, but for a few weeks.
I got banned for inviting random people to the group, so many people reported me, even on discord they DMed Jenn and Tasha!

**Comment by aboutDavid** [comment]
@LankyBox01 Do you even know who got you banned?

**Comment by greasezone** [comment]
Really? Thats crazy. Any idea who is behind it?

**Comment by LankyBox01** [comment]
But most people i invited had requested that i invite them, so that wasn't really fair

**Comment by aboutDavid** [comment]
> Any idea who is behind it?
Yes, that what I mean.

**Comment by greasezone** [comment]
But like who contacted the mods?

**Comment by greasezone** [comment]
https://pigeon-video-chat.glitch.me/

**Comment by LankyBox01** [comment]
On the system evertyone can see which invited which and where

**Comment by greasezone** [comment]
Lets talk here

**Comment by aboutDavid** [comment]
Why cant we chat here: https://pigeon-video-chat.glitch.me/ ?

**Comment by LankyBox01** [comment]
People could see which invited which.
And they tought i was inviting people randomly, but i wasn't, i either got their permission, or they requested i invite them

That was NOT fair!

**Comment by RiversideRocks** [comment]
I was invited randomly.

**Comment by RiversideRocks** [comment]
So was aboutDavid and everyone else.

**Comment by greasezone** [comment]
Really? Come on @LankyBox01

**Comment by LankyBox01** [comment]
They were like 3-4. I did that 4 days ago, the chaos just began yesterday.

YEET

**Comment by LankyBox01** [comment]
Anyways, you get the point

**Comment by greasezone** [comment]
Well I guess you deserved the ban.

**Comment by LankyBox01** [comment]
Yea

**Comment by LankyBox01** [comment]
Are you grease0 in the chat?

**Comment by LankyBox01** [comment]
Hello?

**Comment by greasezone** [comment]
No, I'm not in the chat now.

**Comment by LankyBox01** [comment]
Hmm

**Comment by LankyBox01** [comment]
Where should we chat privately?

**Comment by aboutDavid** [comment]
I think I know

**Comment by LankyBox01** [comment]
?

**Comment by greasezone** [comment]
We could go to the riverside rocks forum

**Comment by greasezone** [comment]
The mods dont care there

**Comment by LankyBox01** [comment]
Wasn't it deleted?

**Comment by greasezone** [comment]
https://riverside.rocks/community

**Comment by LankyBox01** [comment]
Oh

**Comment by aboutDavid** [comment]
You need a Discord account tho.

**Comment by aboutDavid** [comment]
Riverside will add GitHub sign-in later.

**Comment by LankyBox01** [comment]
Yee

**Comment by greasezone** [comment]
Just sign up for discord, like make a throwaway account

**Comment by LankyBox01** [comment]
I was a mod once, but i was falsely accused of being under 12 smh

**Comment by greasezone** [comment]
Odd.

**Comment by greasezone** [comment]
My username there is m4sugar

**Comment by greasezone** [comment]
Are you going to make a thread or should I?

**Comment by LankyBox01** [comment]
Why don't you build a flarum community for PigeonVideo?

**Comment by aboutDavid** [comment]
https://riverside.rocks/community

**Comment by greasezone** [comment]
Flarum sucks, you should use the riveride forum. The UI is better on the riverside forum

**Comment by LankyBox01** [comment]
Nah

**Comment by LankyBox01** [comment]
I hate making new accounts.

**Comment by greasezone** [comment]
I think about david already made a thread I am replying now

**Comment by greasezone** [comment]
I think about david already made a thread I am replying now

**Comment by LankyBox01** [comment]
I said we should chat *privately*

**Comment by greasezone** [comment]
There are PMs there

**Comment by LankyBox01** [comment]
Ugh, you do you. I do me

**Comment by LankyBox01** [comment]
I won't join.

**Comment by greasezone** [comment]
Ok well we are already chatting there 💯 💯 💯 💯 💯 💯 💯 💯

**Comment by greasezone** [comment]
💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯

**Comment by aboutDavid** [comment]
https://pigeonvideoforum.flarum.cloud/

**Comment by aboutDavid** [comment]
Just got it setup!

**Comment by LankyBox01** [comment]
Thank you!

**Comment by aboutDavid** [comment]
Sign up so I can give you admin

**Comment by LankyBox01** [comment]
Could you activate my account. I canno copy-paste links on mobile with a 10-minute-mail

**Comment by LankyBox01** [comment]
LankyBox01

**Comment by aboutDavid** [comment]
Let me see if I can add GitHub login also. I'll activate the account for you.

**Comment by LankyBox01** [comment]
Thank you. Also add discord if possible :P

**Comment by LankyBox01** [comment]
https://pigeonvideoforum.flarum.cloud/u/LankyBox01
Activate me account!

**Comment by aboutDavid** [comment]
Activated! I forgot how to for a moment.

**Comment by LankyBox01** [comment]
Now make me an administrstor..

**Comment by aboutDavid** [comment]
> Now make me an administrstor..

Ok, it's being weird.

**Comment by aboutDavid** [comment]
You should be an admin.

**Comment by LankyBox01** [comment]
I'm not.

**Comment by aboutDavid** [comment]
> I'm not.

I've clicked the button 15 times, let me try again.

**Comment by aboutDavid** [comment]
Try now.

**Comment by LankyBox01** [comment]
Okie works

**Comment by aboutDavid** [comment]
@greasezone did you sign up, if so, check discord!

**Comment by LankyBox01** [comment]
He dd not sign up. I bet he's even seeing this

**Comment by aboutDavid** [comment]
Added Discord login. GitHub's login is being difficult.

**Comment by aboutDavid** [comment]
https://u.aboutdavid.me/La3reAp.png
@LankyBox01 I can't leave the account as it is associated with the email.

**Comment by LankyBox01** [comment]
It's possible to change a single email, right?

**Comment by aboutDavid** [comment]
> It's possible to change a single email, right?

I'm not using multiple emails,

**Comment by LankyBox01** [comment]
I was just.. ugh
Can you make an email on yahoo?

**Comment by aboutDavid** [comment]
> I'm not using multiple emails,

**Comment by LankyBox01** [comment]
No, for the team. Not just.. you

**Comment by LankyBox01** [comment]
PigeonVideo@aboutDavid.me
Is that reasonable?

**Comment by Readtt** [comment]
I had a problem with something and it kicked me from the thing, can you invite me again?

**Comment by aboutDavid** [comment]
> [PigeonVideo@aboutDavid.me](mailto:PigeonVideo@aboutDavid.me)
> Is that reasonable?

I don't even have an email server anymore

**Comment by Readtt** [comment]
Going to profile doesn't seem to work in:
https://pigeonvideoforum.flarum.cloud/u/HK1

**Comment by LankyBox01** [comment]
https://github.com/PigeonVideo/PigeonVideo.github.io/issues/9

**Comment by Readtt** [comment]
?

**Comment by LankyBox01** [comment]
We'll chat there.
